sale_stock_propagate_quantity_diminution/models/sale.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.tools import float_compare
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _log_decrease_ordered_quantity(self, documents, cancel=False):
+        # no log message as we support the feature
+        pass
```

Comment:
Why remove this functionality? In my opinion, the best option is to modify the `exception_on_so` template and remove the part of the message where it says that a manual action is needed, isn't it?